src/m365/todo/commands/task/task-remove.ts

```diff
@@ -0,0 +1,151 @@
+import * as chalk from 'chalk';
+import { Cli, Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  listName?: string;
+  listId?: string;
+  confirm?: boolean;
+}
+
+class TodoTaskRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TASK_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Removes a Task in a Microsoft To Do task list';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
```

Comment:
Since `id` is required, there is no need to track it in the telemetry because it would always show up as set.